returnn/hdf.py

```diff
@@ -293,35 +295,34 @@ def __init__(
         alignment_caches: List[tk.Path],
         allophone_file: tk.Path,
         state_tying_file: tk.Path,
+        data_type: type = np.uint16,
         returnn_root: Optional[tk.Path] = None,
     ):
         """
-        :param list[Path]: alignment_caches
-        :param Path: allophone_file
-        :param Path: state_tying_file
-        :param Optional[Path] returnn_root: file path to the RETURNN repository root folder
+        :param alignment_caches: e.g. output of an AlignmentJob
+        :param allophone_file
+        :param state_tying_file: e.g. output of a DumpStateTyingJob
+        :param returnn_root: file path to the RETURNN repository root folder
```

Comment:
```suggestion
        :param allophone_file: e.g. output of a StoreAllophonesJob
        :param state_tying_file: e.g. output of a DumpStateTyingJob
        :param data_type: type that is used to store the data
        :param returnn_root: file path to the RETURNN repository root folder
```
added new data_type to docstring.